shopinvader_schema_address/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,4 @@
+[ This file must be max 2-3 paragraphs, and is required. ]
```

Comment:
A little description to explain why we created this addon... and you can remove the unsed files

Reply:
Indeed, it is in my todo list